babun-core/plugins/shell-here/install.sh

```diff
@@ -0,0 +1,22 @@
+set -e -f -o pipefail
+source "/usr/local/etc/babun.instance"
+source "$babun_tools/script.sh"
+
+babun_root=$( cygpath -w "/" | sed "s#\\\cygwin##g" )
+name="Babun Shell here"
+cmd="${babun_root}\cygwin\bin\mintty.exe /bin/env CHERE_INVOKING=1 /bin/zsh.exe"
+
+keys=("HKCU\Software\Classes\Directory\Background\shell\babun"
+	"HKCU\Software\Classes\Directory\shell\babun"
+	"HKCU\Software\Classes\Drive\Background\Shell\babun"
+	"HKCU\Software\Classes\Drive\shell\babun")
+
+#start with installing chere
+pact install chere || echo "Installing 'chere' failed"
+
+#install registry keys
+for key in ${keys[*]}
+do 
+	cmd /c "reg" "add" "${key}" "/ve" "/d" "${name}" "/t" "REG_SZ" "/f" || echo "Failed adding ${key}"
+	cmd /c "reg" "add" "${key}\command" "/ve" "/d" "${cmd}" "/t" "REG_EXPAND_SZ" "/f" || echo "Failed adding ${key}"
```

Comment:
Thanks. The plugin looks great!
One more question that came to my mind just now: will the registry modification overwrite keys if they have already existed? 
It's possible to have a couple of babun instances at once (e.g. using the /target option).
We have to think of a way how to make the behavior consistent, or at least give the user a chance to control it.
if it fails without overwriting - it's possible that the user may be locked with a stale state.
Otherwise, the user may destroy existing babun installation, just by making a test and installing the "candidate-release" on a separate drive... don't know which is better...